gpkit/nomials.py

```diff
@@ -430,14 +449,30 @@ def __float__(self):
 
 
 class Posynomial(Signomial):
+    """A Signomial with strictly positive cs
+
+    Arguments
+    ---------
+    Same as Signomial.
+    Note: Posynomial historically supported several different init formats
+          These will be deprecated in the future, replaced with a single
+          __init__ syntax, same as Signomial.
+    """
     pass
 
 
 class Monomial(Posynomial):
-    '''
-    TODO: Add docstring
-    '''
+    """A Posynomial with only one term
+
+    Arguments
+    ---------
+    Same as Signomial.
+    Note: Monomial historically supported several different init formats
+          These will be deprecated in the future, replaced with a single
+          __init__ syntax, same as Signomial.
+    """
     def __rdiv__(self, other):
+        """Divide other by this Monomial"""
```

Comment:
Had a thought -- would it be possible to remove the `elif isinstance(other, Monomial)` case from `Signomial.__div__`, so that we always fall back on `Monomial`'s `rdiv`? That seems conceptually cleaner, but I failed to get Python to fall back on `__rdiv__` in the simple attempts I made.